pypreprocess/reporting/check_preprocessing.py

```diff
@@ -161,31 +153,16 @@ def plot_cv_tc(epi_data, session_ids, subject_id,
             os.makedirs(cache_dir)
         mem = joblib.Memory(cachedir=cache_dir, verbose=5)
         cv = mem.cache(compute_cv)(data, mask_array=mask_array)
+        cv = nibabel.Nifti1Image(cv, affine)
+
+        # XXX nilearn complains about rotations in affine, etc.
+        cv = reorder_img(cv, resample="continuous")
 
         if write_image:
             # write an image
-            nibabel.save(nibabel.Nifti1Image(cv, affine),
-                         os.path.join(_output_dir, 'cv_%s.nii' % session_id))
-            if bg_image == False:
-                try:
-                    viz.plot_map(
-                        cv, affine, threshold=.01, cmap=viz.cm.cold_hot)
-                except IndexError:
-                    print traceback.format_exc()
-            else:
-                if isinstance(bg_image, basestring):
-                    _tmp = nibabel.load(bg_image)
-                    anat, anat_affine = (
-                        _tmp.get_data(),
-                        _tmp.get_affine())
-                else:
-                    anat, anat_affine = data.mean(-1), affine
-                try:
-                    viz.plot_map(
-                        cv, affine, threshold=.01, cmap=viz.cm.cold_hot,
-                             anat=anat, anat_affine=anat_affine)
-                except IndexError:
-                    print traceback.format_exc()
+            cv.to_filename(os.path.join(_output_dir, 'cv_%s.nii' % session_id))
+        plot_stat_map(cv, threshold=.01, cmap=viz.cm.cold_hot, **kwargs)
+
         # compute the time course of cv
```

Comment:
Sounds good. is iit possible to add a test for plot_cv_tc ? I think that this would help for future developments.